### PR TITLE
Table regen

### PR DESCRIPTION
**Pull request by MichalMaler**
Signed-off-by: Michal Maléř <mmaler@redhat.com>

> Read our [Contribution guide](https://github.com/eclipse/che-docs/blob/master/CONTRIBUTING.adoc) before submitting a PR.

### PR Checklist

As the author of this Pull Request I made sure that:

- [ ] `vale` has been run successfully against the PR branch
- [ ] Link checker has been run successfully against the PR branch
- [ ] Documentation describes a scenario that is already covered by QE tests, otherwise an issue has been created and acknowledged by Che QE team
- [ ] Changed article references are updated where they are used (or a redirect has been set up on the docs side):
    - [ ] Dashboard [branding.json](https://github.com/eclipse/che-dashboard/blob/master/src/components/branding/branding.json)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)

### CODE REVIEW

File: modules/installation-guide/examples/system-variables.adoc [hunk 1]
```diff
@@ -226,15 +226,16 @@
...
  `+CHE_WORKSPACE_PROVISION_SECRET_LABELS+`,"`+app.kubernetes.io/part-of=che.eclipse.org,app.kubernetes.io/component=workspace-secret+`","Defines comma-separated list of labels for selecting secrets from a user namespace, which will be mount into workspace containers as a files or env variables. Only secrets that match ALL given labels will be selected." 
- `+CHE_WORKSPACE_DEVFILE_ASYNC_STORAGE_PLUGIN+`,"`+eclipse/che-async-pv-plugin/nightly+`","Plugin is added in case async storage feature will be enabled in workspace config and supported by environment" 
- `+CHE_INFRA_KUBERNETES_ASYNC_STORAGE_IMAGE+`,"`+quay.io/eclipse/che-workspace-data-sync-storage:latest+`","Docker image for the {prod-short} async storage" 
+ `+CHE_WORKSPACE_DEVFILE_ASYNC_STORAGE_PLUGIN+`,"`+eclipse/che-async-pv-plugin/latest+`","Plugin is added in case async storage feature will be enabled in workspace config and supported by environment" 
+ `+CHE_INFRA_KUBERNETES_ASYNC_STORAGE_IMAGE+`,"`+quay.io/eclipse/che-workspace-data-sync-storage:0.0.1+`","Docker image for the {prod-short} async storage"
```
Comment (nickboldt) [inline modules/installation-guide/examples/system-variables.adoc:233]: latest is newer than 0.0.1. Why use old one and have to update it manually? 

https://quay.io/repository/eclipse/che-workspace-data-sync-storage?tag=latest&tab=tags

Comment (themr0c) [reply modules/installation-guide/examples/system-variables.adoc:233]: Then we need to update it in the source in: https://github.com/eclipse/che/blob/7.25.x/assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/multiuser.properties

**Comment by themr0c** [comment]
> Also what is "RedHat Che" ? Is that something like Eclipse Che?

That's exactly the leftover we need to remove.

See:https://github.com/eclipse/che/pull/18938

**Comment by rkratky** [comment]
> What is "RedHat" ? Is that something like "Red Hat" (with space) ?

Yes.

> Also what is "RedHat Che" ? Is that something like Eclipse Che?

No. It's some weird remnant of a past misunderstanding. The purpose of this PR is to get rid of the reference because it was offending people to no end.

**Review by themr0c — APPROVED** [review]

File: modules/installation-guide/examples/system-variables.adoc (hunk 1 above)
Comment (themr0c) [reply modules/installation-guide/examples/system-variables.adoc:233]: @nickboldt target is 7.25.x, are you sure the version should be updated in  https://github.com/eclipse/che/blob/7.25.x/assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/multiuser.properties ?

Comment (nickboldt) [reply modules/installation-guide/examples/system-variables.adoc:233]: no idea, I was just pointing out a discrepancy. @skabashnyuk might be able to help you here

**Review by nickboldt — APPROVED** [review]
LGTM, but not an expert in specific versions vs. `latest`

**Pull request merged**